content/docs/300-on-brand-menu-api/400-reference/300-get-menu-groups-options.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: GET /v1/menus/:menu_id/groups
```

Comment:
```suggestion
title: GET /v1/menus/:menuId/groups
```

(nitpick) We use camelCase in the responses so I'd like to be consistent everywhere. My thinking is that the menu API is mostly going to be consumed by javascript. 

Mind replacing `menu_id` with `menuId` in the few places? 

PR looks great otherwise!